### DISCUSSION TITLE
How do I access the CLI in your dockers?

### DISCUSSION THREAD

**Discussion by darrylrev** [jlesage/docker-avidemux#13] [Q&A] [answered]
I'm trying to create a script for Avidemux, but I can't figure out how to access the CLI. I can open a terminal window in the docker, but I don't see any applications. I've tried using ls and I don't see any applications in the docker at all. Normally I can use a docker exec to reach the application inside the Docker but in this case, it doesn't appear at all. I'm using UnRaid.

Thank you!

**Comment by jlesage** [comment, marked as answer]
I guess you are talking about the Avidemux cli ?  It is located at `/usr/bin/avidemux3_cli` inside the container.

**Comment by hoonlight** [comment]
Hi,

I ran avidemux installed on my host with the following script. 
This is not the CLI version, running the script will run the GUI version, it's just automated.

```
./run_avidemux_template.sh --yes --load "$video" --video-codec copy --audio-codec copy --output-format "$output_ext" --save "$temp_output" --quit
```

[run_avidemux_template.sh](https://github.com/mean00/avidemux2/blob/master/run_avidemux_template.sh)

Is there any way to do this in docker as well? I needed to process a bunch of videos for the purpose of rebuilding the PTS of a video, and this worked fine.

**Reply by hoonlight** [reply]
I modified the startapp.sh inside the container and it works fine.
